cmd/client-fs.go

```diff
@@ -183,6 +183,14 @@ func (f *fsClient) Watch(params watchParams) (*watchObject, *probe.Error) {
 				i, e = os.Stat(event.Path())
 				if e != nil {
 					if os.IsNotExist(e) {
+						if runtime.GOOS == "windows" && event.Event()&notify.Rename != 0 {
```

Comment:
Does this perhaps need to be done outside? because looking for delete event can be done before doing a stat on the path?

Reply:
This is for the case of renaming of a file (notify.Rename). The event (notify.Rename) is generated with the older filename and the newer(changed) filename. We need this check to distinguish between the older & newer names. Please advise.

Reply:
Correct but it's added deep down after os.Stat() why not before?